docs/pages/web/badge.js

```diff
@@ -18,6 +18,20 @@ export default function BadgePage({ generatedDocGen }: {| generatedDocGen: DocGe
         defaultCode={`
 <Text>Update your pronouns in your profile settings <Badge text="New" /></Text>
     `}
+        slimBanner={
+          <SlimBanner
+            iconAccessibilityLabel="Component under experiment"
+            message={`Badge is under an experiment to update its visual design: font size, spacing, and a new information Icon and Tooltip for type="info". Badge changes are active on the Gestalt Docs.`}
```

Comment:
I don't think we'll want to show this yet until we provide the user a way to also see the non-experimental treatment

Reply:
ditto